easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2022.02.0-443-rhel-x86_64.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'Binary'
+
+name = 'RStudio-Server'
+version = '2022.02.0-443'
+versionsuffix = '-rhel-%s' % ARCH
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port 8787 --rsession-which-r=$(which R)
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download2.rstudio.org/server/rhel8/%s' % ARCH]
+sources = ['rstudio-server-rhel-%%(version)s-%s.rpm' % ARCH]
+checksums = ['8eb698ddde2043b9f143296c46a4b1643e7a4b5373740f15e016bf2da1ad6289']
+
+# both rpm2cio (provided by rpm package) and cpio commands are required for installing
+osdependencies = ['cpio', 'rpm']
```

Comment:
I see

```
osdependencies = [
    ('pam-devel', 'libpam0g-dev')
]
```

in e.g. RStudio-Server-1.4.1717-foss-2021a-Java-11-R-4.1.0.eb . Wouldn't this `rpm` installation depend on those too?

Reply:
the rpm only has deps on sqlite and postgresql-libs. sqlite is hard to avoid on an OS these days.
postgresql-libs is optional afaik